corehq/apps/sms/management/commands/migrate_backend_mappings_to_sql.py

```diff
@@ -0,0 +1,48 @@
+from corehq.apps.sms.mixin import BackendMapping
```

Comment:
(not sure if i made this clear but it's ok to merge this at any time and there's no need to coordinate with a deploy - these are management commands that i'll be running manually)